code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -22,6 +22,7 @@
 	var/elimination = 0
 	anchored = TRUE
 	layer = LARGE_MOB_LAYER //Looks weird with them slipping under mineral walls and cameras and shit otherwise
+	mouse_opacity = 2 // Easier to click on in melee, they're giant targets anyway
```

Comment:
I couldn't really tell the difference when testing it with Legion, but just in case I added mouse_opacity 1 back to legion.